### PR TITLE
Solving Voting issue to vote for every page

### PR DESCRIPTION
**Pull request by YoutLone**
### Hello Ama @khinezarthwe I was changed the following in this issue:

- [x] Now, voting issue is working properly for every page.
- [x] When you vote for the first page and show the voting result page and if you want to vote again you can click vote again button .But when you click second page pagination button it will also showed the voting result of second page . That why if you want to vote second page you can click next button .
- [x] If you want to use respective pagination buttons such as 1,2.. you need to click vote again button to show the questions page and now you can go every page by clicking respective pagination buttons and it will show the questions.

_Thank you 🙏🙏for your review🙂🙂🙂_.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,4 +72,4 @@ You can now access the following URLs in your web browser:
 
 9. **DBdiagram**:
 Our Database Design
-![DB](https://github.com/khinezarthwe/mmdt_web_app/assets/5201124/c2d8a65e-1019-4442-8d62-48efe1552033)
+![DB](https://github.com/khinezarthwe/mmdt_web_app/assets/5201124/c2d8a65e-1019-4442-8d62-48efe1552033)
```
Comment (khinezar-mercari) [inline README.md:75]: is there any difference??

File: mmdt/mmdt/settings.py [hunk 2]
```diff
@@ -197,4 +197,4 @@
         'handlers': ['rotate_file'],
         'level': 'INFO',
     },
-}
+}
```
Comment (khinezar-mercari) [inline mmdt/mmdt/settings.py:200]: we better add space line at the end of the file, please do not fix it.

File: mmdt/mmdt/urls.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
```
Comment (khinezar-mercari) [inline mmdt/mmdt/urls.py:17]: if you didn't update `mmdt/mmdt/urls.py`, it is better not to add this line

File: mmdt/polls/models.py [hunk 4]
```diff
@@ -30,4 +30,3 @@ class Choice(models.Model):
 
     def __str__(self):
         return self.choice_text
-
```
Comment (khinezar-mercari) [inline mmdt/polls/models.py:33]: this one too, if we didn't update the file, we better not to commit in PR.

File: README.md (hunk 1 above)
Comment (YoutLone) [reply README.md:75]: Actually no have any different.

**Comment by Pyae24** [comment]
It looks great Ako but I suggest that the voting checks should not disappear on reload when a user has done voting for the first page but instead of submitting the votes for that page, he goes to vote for page 2 and then comes back to page 1.

File: README.md (hunk 1 above)
Comment (khinezarthwe) [reply README.md:75]: Please care not to happen next time and I will remove it.

**Pull request merged**